src/darwin/Framework/CHIP/MTRDeviceControllerDataStore.mm

```diff
@@ -968,6 +989,46 @@ - (void)clearAllStoredAttributes
     return clusterDataToReturn;
 }
 
+// Utility for constructing dictionary of nodeID to cluster data from dictionary of storage keys
+- (nullable NSDictionary<NSNumber *, NSDictionary<MTRClusterPath *, MTRDeviceClusterData *> *> *)getClusterDataFromSecureLocalValues:(NSDictionary<NSString *, id> *)secureLocalValues
```

Comment:
Should this be named `_getClusterDataFromSecureLocalValues`?  I haven't figured out when we have the `_` prefix and when not...

Reply:
I typically add a `_` if it's a method that is only used within the file, and not meant to be exposed. So you're right that this should get a `_` and I missed it. 👍

Reply:
Done.